public/javascripts/SVLabel/src/SVLabel/mission/Mission.js

```diff
@@ -153,6 +153,10 @@ function Mission(parameters) {
             };
             svl.labelCounter.reset();
         }
+
+        if (!svl.onboarding.isOnboarding()){
```

Comment:
This still gives an error.

![image](https://user-images.githubusercontent.com/2873216/27928945-185d1610-625f-11e7-864d-2affb291306b.png)

The reason is `svl.onboarding` doesn't exist is because that's only set within Main.js to activate onboarding. The function call is simply `svl.isOnboarding()`